README.md

```diff
@@ -4,18 +4,18 @@
 
 Brokerless task queue for python based on 0Mq
 
-- This library comes with the benefits of 0Mq
-     - Low Latency
-     - Lightweight
-     - No broker required
+- This benefits of libraty of 0Mq

```

Comment:
@ajaySahani7945 there is a typo, libraty  should be library. This sentence is not actually clear `This benefits of libraty of 0Mq`

Reply:
Ok done...